WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -263,6 +264,23 @@ extension PointOfSaleAggregateModel {
     }
 }
 
+// MARK: - Card payments
+//
+extension PointOfSaleAggregateModel {
+    func collectCashPayment() {
+        // Capture the latest payment state before switching to cash collection
+        // so we can return if the collection is cancelled
+        latestPaymentState = paymentState
+        paymentState = .acceptingCash
+    }
+
+    func cancelCashPayment() {
+        if let latestPaymentState = latestPaymentState {
+            paymentState = latestPaymentState
+        }
+    }
```

Comment:
That makes sense, thanks for the pointer. I've reverted these changes for the time being.

> It would probably be better to cancel the reader preparation when they tap cash payment, then trigger it again if they cancel. I believe that's what's being done on Android too, so keeping platform consistency would be good.

👍 I've added a note to https://github.com/woocommerce/woocommerce-ios/issues/14682 for when implementing collecting and cancelling